app/models/affiliate.rb

```diff
@@ -61,7 +61,7 @@ def make_unavailable(template_ids)
 
   has_many :users, -> { order 'contact_name' }, through: :memberships
   has_many :default_users, class_name: 'User', foreign_key: 'default_affiliate_id', dependent: :nullify
-  has_many :rss_feed_urls, -> { uniq }, through: :rss_feeds
+  has_many :rss_feed_urls, -> { distinct }, through: :rss_feeds
```

Comment:
See comment above re. ignoring other deprecations (but feel free to keep these changes as-is).